easybuild/easyconfigs/q/QUAST/QUAST-5.0.2-foss-2020b-Python-2.7.18.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'QUAST'
+version = '5.0.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ablab/%(namelower)s'
+description = """QUAST evaluates genome assemblies by computing various metrics.
+It works both with and without reference genomes. The tool accepts multiple
+assemblies, thus is suitable for comparison."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('Python', '2.7.18'),
+    ('Perl', '5.32.0'),
+    ('matplotlib', '2.2.5', versionsuffix),
+    ('Java', '11', '', True),
+    ('Boost', '1.74.0'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('simplejson', '3.17.2', {
+        'checksums': ['75ecc79f26d99222a084fbdd1ce5aad3ac3a8bd535cd9059528452da38b68841'],
+    }),
```

Comment:
This is no longer needed, as it is provided by our Python 2.7.18 install: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-10.2.0.eb#L382